src/dotnet/DotNetTopLevelCommandBase.cs

```diff
@@ -42,19 +42,26 @@ public int RunCommand(string[] args)
             }
             catch (KeyNotFoundException)
```

Comment:
Unrelated (pre-existing), but it looks bogus to assume that any KeyNotFoundException in command.Execute() means RequiredCommandNotPassed. Do you agree or am I missing something? If you agree, I'll file a bug.

Reply:
Agreed, the `Execute` call should be outside of this try-catch.

Reply:
Filed #8752